lib/descriptor/type/multisig.js

```diff
@@ -0,0 +1,216 @@
+/*!
+ * multisig.js - multisig descriptor object for bcoin
+ * Copyright (c) 2023, the bcoin developers (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const AbstractDescriptor = require('../abstractdescriptor');
+const common = require('../common');
+const {isType, strip, checkChecksum, scriptContext, types} = common;
+const consensus = require('../../protocol/consensus');
+const {MAX_SCRIPT_PUSH, MAX_MULTISIG_PUBKEYS} = consensus;
+const KeyProvider = require('../keyprovider');
+const Network = require('../../protocol/network');
+const assert = require('bsert');
+
+/**
+ * MultisigDescriptor
+ * Represents a multisig output script.
+ * @see https://github.com/bitcoin/bips/blob/master/bip-0383.mediawiki
+ * @property {String} type
+ * @property {KeyProvider[]} keyProviders
+ * @property {Number} threshold
+ * @property {Boolean} isSorted - true if descriptor is sortedmulti
+ * @property {Network} network
+ * @extends Descriptor
+ */
+
+class MultisigDescriptor extends AbstractDescriptor {
+  /**
+   * @constructor
+   * @param {Object} options
+   */
+
+  constructor(options) {
+    super();
+    this.type = types.MULTI;
+    this.isSorted = false;
```

Comment:
```suggestion
    this.type = types.SORTEDMULTI;
    this.isSorted = true;
```
default value should be sorted

Reply:
resolved